components/logins/src/db.rs

```diff
@@ -752,6 +752,13 @@ impl LoginDb {
         Ok(exists)
     }
 
+    pub fn delete_encrypted_login_data(&self) -> Result<()> {
```

Comment:
I was going to say I don't like the name of this function, and suggested we called it something like "wipe" - and then recalled we already have something named that. Then I discovered `wipe_local` - and ISTM that this is just reimplementing `wipe_local()`? Sorry I didn't realize this before, but unless I'm missing something, it seems `wipe_local()` already does exactly what we need?

Reply:
Ugh, yeah, you're right. The only unique thing this logic does is reset the last sync time which still isn't meaningful enough to warrant these changes. I'm going to close this PR and archive the scrubbing logic ticket. Thanks!